harmonica/equivalent_layer/utils.py

```diff
@@ -28,19 +28,21 @@ def jacobian_numba(coordinates, points, jac, greens_function):
             )
 
 
-@jit(nopython=True)
-def predict_numba(coordinates, points, coeffs, result, greens_function):
+@jit(nopython=True, parallel=True)
+def predict_numba(
+    coordinates, points, coeffs, result, greens_function
+):  # pylint: disable=not-an-iterable
     """
     Calculate the predicted data using numba for speeding things up.
 
-    It works both for Cartesian and spherical coordiantes.
-    We need to pass the corresponding Green's function through the
-    ``greens_function`` argument.
+    It works both for Cartesian and spherical coordiantes. We need to pass the
+    corresponding Green's function through the ``greens_function`` argument.
+    The prediction is run in parallel in order to reduce the computation time.
     """
     east, north, upward = coordinates[:]
     point_east, point_north, point_upward = points[:]
-    for i in range(east.size):
-        for j in range(point_east.size):
+    for i in prange(east.size):
+        for j in prange(point_east.size):
```

Comment:
Might be worth benchmarking if it's better to have a `prange` in the outer `for` only.

Reply:
I haven't seen any significant difference, but I think it's wiser to leave only `prange` in the outer loop only, we will never have that many cores to split the computation of almost one matrix element per core (remind me of reading this same sentence in 30 years from now haha).